websites/U/UKTV/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({
+		clientId: "1002899869599551508",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		browsing: "general.browsing",
+	});
```

Comment:
If you have strings, you should include a multiLang setting and use the Awaited type for strings

Reply:
> If you have strings, you should include a multiLang setting and use the Awaited type for strings

but no 1000 users?

Reply:
That rule is for adding custom strings for said presence, not adding multiLang in general.

Reply:
> That rule is for adding custom strings for said presence, not adding multiLang in general.

ok thx for clearing that up.